src/app/screens/Unlock/index.tsx

```diff
@@ -45,6 +47,7 @@ function Unlock() {
     setLoading(true);
     auth
       .unlock(password, () => {
+        if (navState.action === "unlock") msg.reply({ unlocked: true });
```

Comment:
Shouldn't we return here after sending msg.reply?

Reply:
why return? unlock screen is open via openPrompt screen. when we msg.reply call is already completed and prompt is closed

Reply:
I just think it's a bit cleaner code-wise. Otherwise one could think the other code is still being executed (navigate(), setLoading())...


```suggestion
        if (navState.action === "unlock") {
            msg.reply({ unlocked: true });
            return;
        }
```

Reply:
resolved